mysql-db-init/entrypoint.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+export MYSQLHOST="${MYSQL_HOST}"
+export MYSQLPASSWORD="${MYSQL_SUPER_PASS}"
+
+if [[ -z "${MYSQLHOST}" || -z "${MYSQLPASSWORD}" || -z "${MYSQL_USER}" || -z "${MYSQL_PASS}" || -z "${MYSQL_DB}" ]]; then
+  printf "\e[1;32m%-6s\e[m\n" "environment variables missing ..."
+  exit 1
+fi
+
+until mysqladmin --user=root --password=${MYSQLPASSWORD} --protocol tcp ping -h ${MYSQLHOST} 2>/dev/null
+do
+  printf "\e[1;32m%-6s\e[m\n" "waiting for host '${MYSQLHOST}' ..."
+  sleep 1
+done
+
+user_exists=$(\
+  mysql --user=root \
```

Comment:
Looks ok, just code is duplicated 6 times. Basically we call mysql command with query 6 times. Would be really nice to have function like exec_command(query)
it would be much more readable, but its not required from my side :-)

Reply:
Yeah, it would be nice, but not requirement.